framework/src/modules/interoperability/constants.ts

```diff
@@ -29,6 +29,7 @@ export const MAX_LENGTH_NAME = 40;
 export const MAX_UINT32 = 4294967295;
 export const MAX_UINT64 = BigInt('18446744073709551615'); // BigInt((2 ** 64) - 1) - 1
 export const THRESHOLD_MAINCHAIN = 68;
+export const MESSAGE_TAG_CERTIFICATE = Buffer.from('LSK_CE_', 'ascii');
```

Comment:
why `ascii` here? in the signBLS, it's using `utf8`

Reply:
ah i just followed LIP, but yeah we need to use utf here